packages/bitcore-wallet-service/src/lib/middleware.ts

```diff
@@ -4,11 +4,14 @@ import { formatTimestamp, logger } from './logger';
 type TimedRequest = {
   startTime?: Date;
   walletId?: string;
+  isSupportStaff?: boolean;
 } & express.Request;
 
 function LogObj(logOut: { [key: string]: string }) {
   logger.info(
-    `${logOut.time} |  ${logOut.walletId} | ${logOut.ip} | ${logOut.phase} | ${logOut.took} | ${logOut.method} | ${logOut.status} | ${logOut.url} | ${logOut.openConnections} open`
+    `${logOut.support}${logOut.time} | ${logOut.ip} | ${logOut.userAgent || 'na'}  |  ${logOut.walletId || '-'}  | ${logOut.phase} | ${
```

Comment:
Is there an | (or) operand missing at ```${logOut.support}${logOut.time}```

Reply:
> no. it it is just a prefix to note requests from support
> […](#)
> On Tue, Jul 21, 2020, 13:29 Bernard J Snowden III ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In packages/bitcore-wallet-service/src/lib/middleware.ts <[#2901 (comment)](https://github.com/bitpay/bitcore/pull/2901#discussion_r458230191)>: > } & express.Request; function LogObj(logOut: { [key: string]: string }) { logger.info( - `${logOut.time} | ${logOut.walletId} | ${logOut.ip} | ${logOut.phase} | ${logOut.took} | ${logOut.method} | ${logOut.status} | ${logOut.url} | ${logOut.openConnections} open` + `${logOut.support}${logOut.time} | ${logOut.ip} | ${logOut.userAgent || 'na'} | ${logOut.walletId || '-'} | ${logOut.phase} | ${ Is there an | (or) operand missing at ${logOut.support}${logOut.time}? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#2901 (review)](https://github.com/bitpay/bitcore/pull/2901#pullrequestreview-452630247)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAYEHHS3P2RGE76VYZUTITR4W65NANCNFSM4PDXZKMQ> .



> no. it it is just a prefix to note requests from support
> […](#)
> On Tue, Jul 21, 2020, 13:29 Bernard J Snowden III ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In packages/bitcore-wallet-service/src/lib/middleware.ts <[#2901 (comment)](https://github.com/bitpay/bitcore/pull/2901#discussion_r458230191)>: > } & express.Request; function LogObj(logOut: { [key: string]: string }) { logger.info( - `${logOut.time} | ${logOut.walletId} | ${logOut.ip} | ${logOut.phase} | ${logOut.took} | ${logOut.method} | ${logOut.status} | ${logOut.url} | ${logOut.openConnections} open` + `${logOut.support}${logOut.time} | ${logOut.ip} | ${logOut.userAgent || 'na'} | ${logOut.walletId || '-'} | ${logOut.phase} | ${ Is there an | (or) operand missing at ${logOut.support}${logOut.time}? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#2901 (review)](https://github.com/bitpay/bitcore/pull/2901#pullrequestreview-452630247)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAYEHHS3P2RGE76VYZUTITR4W65NANCNFSM4PDXZKMQ> .

Ok, ACK!